packages/terra-table/src/terra-dev-site/doc/table/HeaderCell.5.doc.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import DocTemplate from 'terra-doc-template';
+import ReadMe from '../../../../docs/TableHeaderCell.md';
+import { name } from '../../../../package.json';
+
+/* eslint-disable import/no-webpack-loader-syntax, import/first, import/extensions, import/no-unresolved, import/no-duplicates */
+// Component Source
+import TableHeaderCellSrc from '!raw-loader!../../../../src/TableHeaderCell';
+
+import HeaderCellExample from '../example/TableHeaderCell';
+import HeaderCellExampleSrc from '!raw-loader!../../../../src/terra-dev-site/doc/example/TableHeaderCell';
+/* eslint-enable import/no-webpack-loader-syntax, import/first, import/extensions, import/no-unresolved, import/no-duplicates */
+
+const DocPage = () => (
+  <DocTemplate
+    packageName={name}
+    readme={ReadMe}
+    srcPath={`https://github.com/cerner/terra-core/tree/master/packages/${name}`}
+    examples={[
+      {
+        example: <HeaderCellExample />,
+        source: HeaderCellExampleSrc,
```

Comment:
We should add a [description](https://github.com/cerner/terra-core/blob/master/packages/terra-doc-template/src/DocTemplate.jsx#L39) here to indicate sort functionality is left to the consumers and maybe link the doc you wrote for this

Reply:
Updated in readme.